addon/components/frost-modal-dialog.js

```diff
@@ -1,14 +1,32 @@
+/* global Ps */
 import Ember from 'ember'
-import layout from '../templates/components/frost-modal-dialog'
 import computed from 'ember-computed-decorators'
+import PropTypeMixin, {PropTypes} from 'ember-prop-types'
 
 const {Component, assert} = Ember
 
-export default Component.extend({
-  layout: layout,
+export default Component.extend(PropTypeMixin, {
```

Comment:
Based on the note above about not needing to add PropTypeMixin, if you want, you can change this declaration to:

`export default Component.extend({`